### DISCUSSION TITLE
How to change the password generator dictionary to other language?

### DISCUSSION THREAD

**Discussion by karelion-s** [dani-garcia/vaultwarden#4380] [Q&A]
Where can I change, mod , add dictionaries for the password generator?

**Comment by stefan0xC** [comment]
It's hard coded into the web-vault (and other clients). So you'd have to replace the [word list](https://github.com/bitwarden/clients/blob/web-v2024.2.2/libs/common/src/platform/misc/wordlist.ts) with your own and compile it yourself.
